ui/app/pages/send/send.component.js

```diff
@@ -314,19 +325,24 @@ export default class SendTransactionScreen extends Component {
         onReset={() => this.props.updateSendTo('', '')}
         updateEnsResolution={this.props.updateSendEnsResolution}
         updateEnsResolutionError={this.props.updateSendEnsResolutionError}
+        internalSearch={internalSearch}
       />
     )
   }
 
   renderAddRecipient() {
-    const { toError } = this.state
+    const { toError, toWarning } = this.state
     return (
       <AddRecipient
         updateGas={({ to, amount, data } = {}) =>
           this.updateGas({ to, amount, data })
         }
         query={this.state.query}
         toError={toError}
+        toWarning={toWarning}
```

Comment:
This `toWarning` property does not appear to be used by the `AddRecipient` component, even though it is passed to it here.

Reply:
Fixed in: 4b04a88